[libxlsxwriter] build failure if pkgconf is installed first

### Operating system

Windows

### Compiler

MSVC

### Steps to reproduce the behavior

```Shell
git switch -d origin/master
.\vcpkg.exe x-ci-clean
.\vcpkg.exe install libxlsxwriter # succeeds
.\vcpkg.exe remove libxlsxwriter
.\vcpkg.exe install pkgconf
.\vcpkg.exe install libxlsxwriter --binarysource clear # failure repros
```

### Failure logs


```
-- Using cached jmcnamara-libxlsxwriter-v1.1.9.tar.gz
-- Extracting source D:/vcpkg-downloads/jmcnamara-libxlsxwriter-v1.1.9.tar.gz
-- Applying patch dependencies.diff
-- Using source at D:/vcpkg/buildtrees/libxlsxwriter/src/v1.1.9-4b2c6b8898.clean
-- Found external ninja('1.12.1').
-- Configuring x64-windows
-- Building x64-windows-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: "C:/Program Files/CMake/bin/cmake.exe" --build . --config Debug --target install -- -v -j25
    Working Directory: D:/vcpkg/buildtrees/libxlsxwriter/x64-windows-dbg
    See logs for more information:
      D:\vcpkg\buildtrees\libxlsxwriter\install-x64-windows-dbg-out.log

Call Stack (most recent call first):
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/libxlsxwriter/portfile.cmake:24 (vcpkg_cmake_install)
  scripts/ports.cmake:196 (include)



```

<details><summary>D:\vcpkg\buildtrees\libxlsxwriter\install-x64-windows-dbg-out.log</summary>

```
Change Dir: 'D:/vcpkg/buildtrees/libxlsxwriter/x64-windows-dbg'

Run Build Command(s): "C:/Program Files/Microsoft Visual Studio/2022/Enterprise/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" -v -v -j25 install
[1/30] C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\cl.exe   -DNOCRYPT -DNOUNCRYPT -DUSE_SYSTEM_MINIZIP -D_CRT_SECURE_NO_WARNINGS -Dxlsxwriter_EXPORTS -ID:\vcpkg\installed\x64-windows\debug\..\include -ID:\vcpkg\installed\x64-windows\debug\..\include\minizip -ID:\vcpkg\installed\x64-windows\debug\..\include\.. -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include\xlsxwriter /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /showIncludes /FoCMakeFiles\xlsxwriter.dir\src\rich_value_types.c.obj /FdCMakeFiles\xlsxwriter.dir\ /FS -c D:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\src\rich_value_types.c
[2/30] C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\cl.exe   -DNOCRYPT -DNOUNCRYPT -DUSE_SYSTEM_MINIZIP -D_CRT_SECURE_NO_WARNINGS -Dxlsxwriter_EXPORTS -ID:\vcpkg\installed\x64-windows\debug\..\include -ID:\vcpkg\installed\x64-windows\debug\..\include\minizip -ID:\vcpkg\installed\x64-windows\debug\..\include\.. -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include\xlsxwriter /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /showIncludes /FoCMakeFiles\xlsxwriter.dir\src\relationships.c.obj /FdCMakeFiles\xlsxwriter.dir\ /FS -c D:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\src\relationships.c
[3/30] C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\cl.exe   -DNOCRYPT -DNOUNCRYPT -DUSE_SYSTEM_MINIZIP -D_CRT_SECURE_NO_WARNINGS -Dxlsxwriter_EXPORTS -ID:\vcpkg\installed\x64-windows\debug\..\include -ID:\vcpkg\installed\x64-windows\debug\..\include\minizip -ID:\vcpkg\installed\x64-windows\debug\..\include\.. -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include\xlsxwriter /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /showIncludes /FoCMakeFiles\xlsxwriter.dir\src\custom.c.obj /FdCMakeFiles\xlsxwriter.dir\ /FS -c D:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\src\custom.c
[4/30] C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\cl.exe   -DNOCRYPT -DNOUNCRYPT -DUSE_SYSTEM_MINIZIP -D_CRT_SECURE_NO_WARNINGS -Dxlsxwriter_EXPORTS -ID:\vcpkg\installed\x64-windows\debug\..\include -ID:\vcpkg\installed\x64-windows\debug\..\include\minizip -ID:\vcpkg\installed\x64-windows\debug\..\include\.. -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include\xlsxwriter /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /showIncludes /FoCMakeFiles\xlsxwriter.dir\src\app.c.obj /FdCMakeFiles\xlsxwriter.dir\ /FS -c D:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\src\app.c
[5/30] C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\cl.exe   -DNOCRYPT -DNOUNCRYPT -DUSE_SYSTEM_MINIZIP -D_CRT_SECURE_NO_WARNINGS -Dxlsxwriter_EXPORTS -ID:\vcpkg\installed\x64-windows\debug\..\include -ID:\vcpkg\installed\x64-windows\debug\..\include\minizip -ID:\vcpkg\installed\x64-windows\debug\..\include\.. -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include\xlsxwriter /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /showIncludes /FoCMakeFiles\xlsxwriter.dir\src\rich_value_structure.c.obj /FdCMakeFiles\xlsxwriter.dir\ /FS -c D:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\src\rich_value_structure.c
[6/30] C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\cl.exe   -DNOCRYPT -DNOUNCRYPT -DUSE_SYSTEM_MINIZIP -D_CRT_SECURE_NO_WARNINGS -Dxlsxwriter_EXPORTS -ID:\vcpkg\installed\x64-windows\debug\..\include -ID:\vcpkg\installed\x64-windows\debug\..\include\minizip -ID:\vcpkg\installed\x64-windows\debug\..\include\.. -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include\xlsxwriter /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /showIncludes /FoCMakeFiles\xlsxwriter.dir\src\content_types.c.obj /FdCMakeFiles\xlsxwriter.dir\ /FS -c D:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\src\content_types.c
[7/30] C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\cl.exe   -DNOCRYPT -DNOUNCRYPT -DUSE_SYSTEM_MINIZIP -D_CRT_SECURE_NO_WARNINGS -Dxlsxwriter_EXPORTS -ID:\vcpkg\installed\x64-windows\debug\..\include -ID:\vcpkg\installed\x64-windows\debug\..\include\minizip -ID:\vcpkg\installed\x64-windows\debug\..\include\.. -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include\xlsxwriter /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /showIncludes /FoCMakeFiles\xlsxwriter.dir\src\hash_table.c.obj /FdCMakeFiles\xlsxwriter.dir\ /FS -c D:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\src\hash_table.c
[8/30] C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\cl.exe   -DNOCRYPT -DNOUNCRYPT -DUSE_SYSTEM_MINIZIP -D_CRT_SECURE_NO_WARNINGS -Dxlsxwriter_EXPORTS -ID:\vcpkg\installed\x64-windows\debug\..\include -ID:\vcpkg\installed\x64-windows\debug\..\include\minizip -ID:\vcpkg\installed\x64-windows\debug\..\include\.. -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include\xlsxwriter /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /showIncludes /FoCMakeFiles\xlsxwriter.dir\src\metadata.c.obj /FdCMakeFiles\xlsxwriter.dir\ /FS -c D:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\src\metadata.c
[9/30] C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\cl.exe   -DNOCRYPT -DNOUNCRYPT -DUSE_SYSTEM_MINIZIP -D_CRT_SECURE_NO_WARNINGS -Dxlsxwriter_EXPORTS -ID:\vcpkg\installed\x64-windows\debug\..\include -ID:\vcpkg\installed\x64-windows\debug\..\include\minizip -ID:\vcpkg\installed\x64-windows\debug\..\include\.. -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include\xlsxwriter /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /showIncludes /FoCMakeFiles\xlsxwriter.dir\src\theme.c.obj /FdCMakeFiles\xlsxwriter.dir\ /FS -c D:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\src\theme.c
[10/30] C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\cl.exe   -DNOCRYPT -DNOUNCRYPT -DUSE_SYSTEM_MINIZIP -D_CRT_SECURE_NO_WARNINGS -Dxlsxwriter_EXPORTS -ID:\vcpkg\installed\x64-windows\debug\..\include -ID:\vcpkg\installed\x64-windows\debug\..\include\minizip -ID:\vcpkg\installed\x64-windows\debug\..\include\.. -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include\xlsxwriter /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /showIncludes /FoCMakeFiles\xlsxwriter.dir\src\format.c.obj /FdCMakeFiles\xlsxwriter.dir\ /FS -c D:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\src\format.c
[11/30] C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\cl.exe   -DNOCRYPT -DNOUNCRYPT -DUSE_SYSTEM_MINIZIP -D_CRT_SECURE_NO_WARNINGS -Dxlsxwriter_EXPORTS -ID:\vcpkg\installed\x64-windows\debug\..\include -ID:\vcpkg\installed\x64-windows\debug\..\include\minizip -ID:\vcpkg\installed\x64-windows\debug\..\include\.. -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include\xlsxwriter /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /showIncludes /FoCMakeFiles\xlsxwriter.dir\src\rich_value_rel.c.obj /FdCMakeFiles\xlsxwriter.dir\ /FS -c D:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\src\rich_value_rel.c
[12/30] C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\cl.exe   -DNOCRYPT -DNOUNCRYPT -DUSE_SYSTEM_MINIZIP -D_CRT_SECURE_NO_WARNINGS -Dxlsxwriter_EXPORTS -ID:\vcpkg\installed\x64-windows\debug\..\include -ID:\vcpkg\installed\x64-windows\debug\..\include\minizip -ID:\vcpkg\installed\x64-windows\debug\..\include\.. -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include\xlsxwriter /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /showIncludes /FoCMakeFiles\xlsxwriter.dir\src\shared_strings.c.obj /FdCMakeFiles\xlsxwriter.dir\ /FS -c D:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\src\shared_strings.c
[13/30] C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\cl.exe   -DNOCRYPT -DNOUNCRYPT -DUSE_SYSTEM_MINIZIP -D_CRT_SECURE_NO_WARNINGS -Dxlsxwriter_EXPORTS -ID:\vcpkg\installed\x64-windows\debug\..\include -ID:\vcpkg\installed\x64-windows\debug\..\include\minizip -ID:\vcpkg\installed\x64-windows\debug\..\include\.. -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include\xlsxwriter /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /showIncludes /FoCMakeFiles\xlsxwriter.dir\src\chartsheet.c.obj /FdCMakeFiles\xlsxwriter.dir\ /FS -c D:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\src\chartsheet.c
[14/30] C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\cl.exe   -DNOCRYPT -DNOUNCRYPT -DUSE_SYSTEM_MINIZIP -D_CRT_SECURE_NO_WARNINGS -Dxlsxwriter_EXPORTS -ID:\vcpkg\installed\x64-windows\debug\..\include -ID:\vcpkg\installed\x64-windows\debug\..\include\minizip -ID:\vcpkg\installed\x64-windows\debug\..\include\.. -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include\xlsxwriter /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /showIncludes /FoCMakeFiles\xlsxwriter.dir\src\rich_value.c.obj /FdCMakeFiles\xlsxwriter.dir\ /FS -c D:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\src\rich_value.c
[15/30] C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\cl.exe   -DNOCRYPT -DNOUNCRYPT -DUSE_SYSTEM_MINIZIP -D_CRT_SECURE_NO_WARNINGS -Dxlsxwriter_EXPORTS -ID:\vcpkg\installed\x64-windows\debug\..\include -ID:\vcpkg\installed\x64-windows\debug\..\include\minizip -ID:\vcpkg\installed\x64-windows\debug\..\include\.. -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include\xlsxwriter /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /showIncludes /FoCMakeFiles\xlsxwriter.dir\src\styles.c.obj /FdCMakeFiles\xlsxwriter.dir\ /FS -c D:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\src\styles.c
[16/30] C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\cl.exe   -DNOCRYPT -DNOUNCRYPT -DUSE_SYSTEM_MINIZIP -D_CRT_SECURE_NO_WARNINGS -Dxlsxwriter_EXPORTS -ID:\vcpkg\installed\x64-windows\debug\..\include -ID:\vcpkg\installed\x64-windows\debug\..\include\minizip -ID:\vcpkg\installed\x64-windows\debug\..\include\.. -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include\xlsxwriter /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /showIncludes /FoCMakeFiles\xlsxwriter.dir\src\core.c.obj /FdCMakeFiles\xlsxwriter.dir\ /FS -c D:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\src\core.c
[17/30] C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\cl.exe   -DNOCRYPT -DNOUNCRYPT -DUSE_SYSTEM_MINIZIP -D_CRT_SECURE_NO_WARNINGS -Dxlsxwriter_EXPORTS -ID:\vcpkg\installed\x64-windows\debug\..\include -ID:\vcpkg\installed\x64-windows\debug\..\include\minizip -ID:\vcpkg\installed\x64-windows\debug\..\include\.. -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include\xlsxwriter /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /showIncludes /FoCMakeFiles\xlsxwriter.dir\src\comment.c.obj /FdCMakeFiles\xlsxwriter.dir\ /FS -c D:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\src\comment.c
[18/30] C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\cl.exe   -DNOCRYPT -DNOUNCRYPT -DUSE_SYSTEM_MINIZIP -D_CRT_SECURE_NO_WARNINGS -Dxlsxwriter_EXPORTS -ID:\vcpkg\installed\x64-windows\debug\..\include -ID:\vcpkg\installed\x64-windows\debug\..\include\minizip -ID:\vcpkg\installed\x64-windows\debug\..\include\.. -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include\xlsxwriter /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /showIncludes /FoCMakeFiles\xlsxwriter.dir\src\vml.c.obj /FdCMakeFiles\xlsxwriter.dir\ /FS -c D:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\src\vml.c
[19/30] C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\cl.exe   -DNOCRYPT -DNOUNCRYPT -DUSE_SYSTEM_MINIZIP -D_CRT_SECURE_NO_WARNINGS -Dxlsxwriter_EXPORTS -ID:\vcpkg\installed\x64-windows\debug\..\include -ID:\vcpkg\installed\x64-windows\debug\..\include\minizip -ID:\vcpkg\installed\x64-windows\debug\..\include\.. -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include\xlsxwriter /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /showIncludes /FoCMakeFiles\xlsxwriter.dir\src\table.c.obj /FdCMakeFiles\xlsxwriter.dir\ /FS -c D:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\src\table.c
[20/30] C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\cl.exe   -DNOCRYPT -DNOUNCRYPT -DUSE_SYSTEM_MINIZIP -D_CRT_SECURE_NO_WARNINGS -Dxlsxwriter_EXPORTS -ID:\vcpkg\installed\x64-windows\debug\..\include -ID:\vcpkg\installed\x64-windows\debug\..\include\minizip -ID:\vcpkg\installed\x64-windows\debug\..\include\.. -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include\xlsxwriter /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /showIncludes /FoCMakeFiles\xlsxwriter.dir\src\drawing.c.obj /FdCMakeFiles\xlsxwriter.dir\ /FS -c D:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\src\drawing.c
[21/30] C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\cl.exe   -DNOCRYPT -DNOUNCRYPT -DUSE_SYSTEM_MINIZIP -D_CRT_SECURE_NO_WARNINGS -Dxlsxwriter_EXPORTS -ID:\vcpkg\installed\x64-windows\debug\..\include -ID:\vcpkg\installed\x64-windows\debug\..\include\minizip -ID:\vcpkg\installed\x64-windows\debug\..\include\.. -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include\xlsxwriter /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /showIncludes /FoCMakeFiles\xlsxwriter.dir\src\chart.c.obj /FdCMakeFiles\xlsxwriter.dir\ /FS -c D:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\src\chart.c
[22/30] C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\cl.exe   -DNOCRYPT -DNOUNCRYPT -DUSE_SYSTEM_MINIZIP -D_CRT_SECURE_NO_WARNINGS -Dxlsxwriter_EXPORTS -ID:\vcpkg\installed\x64-windows\debug\..\include -ID:\vcpkg\installed\x64-windows\debug\..\include\minizip -ID:\vcpkg\installed\x64-windows\debug\..\include\.. -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include\xlsxwriter /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /showIncludes /FoCMakeFiles\xlsxwriter.dir\src\workbook.c.obj /FdCMakeFiles\xlsxwriter.dir\ /FS -c D:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\src\workbook.c
[23/30] C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\cl.exe   -DNOCRYPT -DNOUNCRYPT -DUSE_SYSTEM_MINIZIP -D_CRT_SECURE_NO_WARNINGS -Dxlsxwriter_EXPORTS -ID:\vcpkg\installed\x64-windows\debug\..\include -ID:\vcpkg\installed\x64-windows\debug\..\include\minizip -ID:\vcpkg\installed\x64-windows\debug\..\include\.. -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include\xlsxwriter /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /showIncludes /FoCMakeFiles\xlsxwriter.dir\src\xmlwriter.c.obj /FdCMakeFiles\xlsxwriter.dir\ /FS -c D:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\src\xmlwriter.c
[24/30] C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\cl.exe   -DNOCRYPT -DNOUNCRYPT -DUSE_SYSTEM_MINIZIP -D_CRT_SECURE_NO_WARNINGS -Dxlsxwriter_EXPORTS -ID:\vcpkg\installed\x64-windows\debug\..\include -ID:\vcpkg\installed\x64-windows\debug\..\include\minizip -ID:\vcpkg\installed\x64-windows\debug\..\include\.. -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include\xlsxwriter /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /showIncludes /FoCMakeFiles\xlsxwriter.dir\third_party\tmpfileplus\tmpfileplus.c.obj /FdCMakeFiles\xlsxwriter.dir\ /FS -c D:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\third_party\tmpfileplus\tmpfileplus.c
[25/30] C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\cl.exe   -DNOCRYPT -DNOUNCRYPT -DUSE_SYSTEM_MINIZIP -D_CRT_SECURE_NO_WARNINGS -Dxlsxwriter_EXPORTS -ID:\vcpkg\installed\x64-windows\debug\..\include -ID:\vcpkg\installed\x64-windows\debug\..\include\minizip -ID:\vcpkg\installed\x64-windows\debug\..\include\.. -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include\xlsxwriter /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /showIncludes /FoCMakeFiles\xlsxwriter.dir\third_party\md5\md5.c.obj /FdCMakeFiles\xlsxwriter.dir\ /FS -c D:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\third_party\md5\md5.c
[26/30] C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\cl.exe   -DNOCRYPT -DNOUNCRYPT -DUSE_SYSTEM_MINIZIP -D_CRT_SECURE_NO_WARNINGS -Dxlsxwriter_EXPORTS -ID:\vcpkg\installed\x64-windows\debug\..\include -ID:\vcpkg\installed\x64-windows\debug\..\include\minizip -ID:\vcpkg\installed\x64-windows\debug\..\include\.. -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include\xlsxwriter /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /showIncludes /FoCMakeFiles\xlsxwriter.dir\src\worksheet.c.obj /FdCMakeFiles\xlsxwriter.dir\ /FS -c D:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\src\worksheet.c
[27/30] C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\cl.exe   -DNOCRYPT -DNOUNCRYPT -DUSE_SYSTEM_MINIZIP -D_CRT_SECURE_NO_WARNINGS -Dxlsxwriter_EXPORTS -ID:\vcpkg\installed\x64-windows\debug\..\include -ID:\vcpkg\installed\x64-windows\debug\..\include\minizip -ID:\vcpkg\installed\x64-windows\debug\..\include\.. -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include\xlsxwriter /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /showIncludes /FoCMakeFiles\xlsxwriter.dir\src\utility.c.obj /FdCMakeFiles\xlsxwriter.dir\ /FS -c D:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\src\utility.c
[28/30] C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\cl.exe   -DNOCRYPT -DNOUNCRYPT -DUSE_SYSTEM_MINIZIP -D_CRT_SECURE_NO_WARNINGS -Dxlsxwriter_EXPORTS -ID:\vcpkg\installed\x64-windows\debug\..\include -ID:\vcpkg\installed\x64-windows\debug\..\include\minizip -ID:\vcpkg\installed\x64-windows\debug\..\include\.. -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include -ID:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\include\xlsxwriter /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  /utf-8 /showIncludes /FoCMakeFiles\xlsxwriter.dir\src\packager.c.obj /FdCMakeFiles\xlsxwriter.dir\ /FS -c D:\vcpkg\buildtrees\libxlsxwriter\src\v1.1.9-4b2c6b8898.clean\src\packager.c
[29/30] C:\Windows\system32\cmd.exe /C "C:\Windows\system32\cmd.exe /C ""C:\Program Files\CMake\bin\cmake.exe" -E __create_def D:\vcpkg\buildtrees\libxlsxwriter\x64-windows-dbg\CMakeFiles\xlsxwriter.dir\.\exports.def D:\vcpkg\buildtrees\libxlsxwriter\x64-windows-dbg\CMakeFiles\xlsxwriter.dir\.\exports.def.objs && cd D:\vcpkg\buildtrees\libxlsxwriter\x64-windows-dbg" && "C:\Program Files\CMake\bin\cmake.exe" -E vs_link_dll --msvc-ver=1942 --intdir=CMakeFiles\xlsxwriter.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100261~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100261~1.0\x64\mt.exe --manifests  -- C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\link.exe  CMakeFiles\xlsxwriter.dir\src\app.c.obj CMakeFiles\xlsxwriter.dir\src\chart.c.obj CMakeFiles\xlsxwriter.dir\src\chartsheet.c.obj CMakeFiles\xlsxwriter.dir\src\comment.c.obj CMakeFiles\xlsxwriter.dir\src\content_types.c.obj CMakeFiles\xlsxwriter.dir\src\core.c.obj CMakeFiles\xlsxwriter.dir\src\custom.c.obj CMakeFiles\xlsxwriter.dir\src\drawing.c.obj CMakeFiles\xlsxwriter.dir\src\format.c.obj CMakeFiles\xlsxwriter.dir\src\hash_table.c.obj CMakeFiles\xlsxwriter.dir\src\metadata.c.obj CMakeFiles\xlsxwriter.dir\src\packager.c.obj CMakeFiles\xlsxwriter.dir\src\relationships.c.obj CMakeFiles\xlsxwriter.dir\src\rich_value.c.obj CMakeFiles\xlsxwriter.dir\src\rich_value_rel.c.obj CMakeFiles\xlsxwriter.dir\src\rich_value_structure.c.obj CMakeFiles\xlsxwriter.dir\src\rich_value_types.c.obj CMakeFiles\xlsxwriter.dir\src\shared_strings.c.obj CMakeFiles\xlsxwriter.dir\src\styles.c.obj CMakeFiles\xlsxwriter.dir\src\table.c.obj CMakeFiles\xlsxwriter.dir\src\theme.c.obj CMakeFiles\xlsxwriter.dir\src\utility.c.obj CMakeFiles\xlsxwriter.dir\src\vml.c.obj CMakeFiles\xlsxwriter.dir\src\workbook.c.obj CMakeFiles\xlsxwriter.dir\src\worksheet.c.obj CMakeFiles\xlsxwriter.dir\src\xmlwriter.c.obj CMakeFiles\xlsxwriter.dir\third_party\tmpfileplus\tmpfileplus.c.obj CMakeFiles\xlsxwriter.dir\third_party\md5\md5.c.obj  /out:xlsxwriter.dll /implib:xlsxwriter.lib /pdb:xlsxwriter.pdb /dll /version:0.0 /machine:x64 /nologo    /debug /INCREMENTAL  /DEF:CMakeFiles\xlsxwriter.dir\.\exports.def  -LD:/vcpkg/installed/x64-windows/debug/lib  -lzlibd  -LD:/vcpkg/installed/x64-windows/debug/lib  -lminizip  kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib && cd ."
FAILED: xlsxwriter.dll xlsxwriter.lib 
C:\Windows\system32\cmd.exe /C "C:\Windows\system32\cmd.exe /C ""C:\Program Files\CMake\bin\cmake.exe" -E __create_def D:\vcpkg\buildtrees\libxlsxwriter\x64-windows-dbg\CMakeFiles\xlsxwriter.dir\.\exports.def D:\vcpkg\buildtrees\libxlsxwriter\x64-windows-dbg\CMakeFiles\xlsxwriter.dir\.\exports.def.objs && cd D:\vcpkg\buildtrees\libxlsxwriter\x64-windows-dbg" && "C:\Program Files\CMake\bin\cmake.exe" -E vs_link_dll --msvc-ver=1942 --intdir=CMakeFiles\xlsxwriter.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100261~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100261~1.0\x64\mt.exe --manifests  -- C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\link.exe  CMakeFiles\xlsxwriter.dir\src\app.c.obj CMakeFiles\xlsxwriter.dir\src\chart.c.obj CMakeFiles\xlsxwriter.dir\src\chartsheet.c.obj CMakeFiles\xlsxwriter.dir\src\comment.c.obj CMakeFiles\xlsxwriter.dir\src\content_types.c.obj CMakeFiles\xlsxwriter.dir\src\core.c.obj CMakeFiles\xlsxwriter.dir\src\custom.c.obj CMakeFiles\xlsxwriter.dir\src\drawing.c.obj CMakeFiles\xlsxwriter.dir\src\format.c.obj CMakeFiles\xlsxwriter.dir\src\hash_table.c.obj CMakeFiles\xlsxwriter.dir\src\metadata.c.obj CMakeFiles\xlsxwriter.dir\src\packager.c.obj CMakeFiles\xlsxwriter.dir\src\relationships.c.obj CMakeFiles\xlsxwriter.dir\src\rich_value.c.obj CMakeFiles\xlsxwriter.dir\src\rich_value_rel.c.obj CMakeFiles\xlsxwriter.dir\src\rich_value_structure.c.obj CMakeFiles\xlsxwriter.dir\src\rich_value_types.c.obj CMakeFiles\xlsxwriter.dir\src\shared_strings.c.obj CMakeFiles\xlsxwriter.dir\src\styles.c.obj CMakeFiles\xlsxwriter.dir\src\table.c.obj CMakeFiles\xlsxwriter.dir\src\theme.c.obj CMakeFiles\xlsxwriter.dir\src\utility.c.obj CMakeFiles\xlsxwriter.dir\src\vml.c.obj CMakeFiles\xlsxwriter.dir\src\workbook.c.obj CMakeFiles\xlsxwriter.dir\src\worksheet.c.obj CMakeFiles\xlsxwriter.dir\src\xmlwriter.c.obj CMakeFiles\xlsxwriter.dir\third_party\tmpfileplus\tmpfileplus.c.obj CMakeFiles\xlsxwriter.dir\third_party\md5\md5.c.obj  /out:xlsxwriter.dll /implib:xlsxwriter.lib /pdb:xlsxwriter.pdb /dll /version:0.0 /machine:x64 /nologo    /debug /INCREMENTAL  /DEF:CMakeFiles\xlsxwriter.dir\.\exports.def  -LD:/vcpkg/installed/x64-windows/debug/lib  -lzlibd  -LD:/vcpkg/installed/x64-windows/debug/lib  -lminizip  kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib && cd ."
LINK Pass 1: command "C:\PROGRA~1\MIB055~1\2022\ENTERP~1\VC\Tools\MSVC\1442~1.344\bin\Hostx64\x64\link.exe CMakeFiles\xlsxwriter.dir\src\app.c.obj CMakeFiles\xlsxwriter.dir\src\chart.c.obj CMakeFiles\xlsxwriter.dir\src\chartsheet.c.obj CMakeFiles\xlsxwriter.dir\src\comment.c.obj CMakeFiles\xlsxwriter.dir\src\content_types.c.obj CMakeFiles\xlsxwriter.dir\src\core.c.obj CMakeFiles\xlsxwriter.dir\src\custom.c.obj CMakeFiles\xlsxwriter.dir\src\drawing.c.obj CMakeFiles\xlsxwriter.dir\src\format.c.obj CMakeFiles\xlsxwriter.dir\src\hash_table.c.obj CMakeFiles\xlsxwriter.dir\src\metadata.c.obj CMakeFiles\xlsxwriter.dir\src\packager.c.obj CMakeFiles\xlsxwriter.dir\src\relationships.c.obj CMakeFiles\xlsxwriter.dir\src\rich_value.c.obj CMakeFiles\xlsxwriter.dir\src\rich_value_rel.c.obj CMakeFiles\xlsxwriter.dir\src\rich_value_structure.c.obj CMakeFiles\xlsxwriter.dir\src\rich_value_types.c.obj CMakeFiles\xlsxwriter.dir\src\shared_strings.c.obj CMakeFiles\xlsxwriter.dir\src\styles.c.obj CMakeFiles\xlsxwriter.dir\src\table.c.obj CMakeFiles\xlsxwriter.dir\src\theme.c.obj CMakeFiles\xlsxwriter.dir\src\utility.c.obj CMakeFiles\xlsxwriter.dir\src\vml.c.obj CMakeFiles\xlsxwriter.dir\src\workbook.c.obj CMakeFiles\xlsxwriter.dir\src\worksheet.c.obj CMakeFiles\xlsxwriter.dir\src\xmlwriter.c.obj CMakeFiles\xlsxwriter.dir\third_party\tmpfileplus\tmpfileplus.c.obj CMakeFiles\xlsxwriter.dir\third_party\md5\md5.c.obj /out:xlsxwriter.dll /implib:xlsxwriter.lib /pdb:xlsxwriter.pdb /dll /version:0.0 /machine:x64 /nologo /debug /INCREMENTAL /DEF:CMakeFiles\xlsxwriter.dir\.\exports.def -LD:/vcpkg/installed/x64-windows/debug/lib -lzlibd -LD:/vcpkg/installed/x64-windows/debug/lib -lminizip kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST /MANIFESTFILE:CMakeFiles\xlsxwriter.dir/intermediate.manifest CMakeFiles\xlsxwriter.dir/manifest.res" failed (exit code 1120) with the following output:
LINK : warning LNK4044: unrecognized option '/LD:/vcpkg/installed/x64-windows/debug/lib'; ignored
LINK : warning LNK4044: unrecognized option '/lzlibd'; ignored
LINK : warning LNK4044: unrecognized option '/LD:/vcpkg/installed/x64-windows/debug/lib'; ignored
LINK : warning LNK4044: unrecognized option '/lminizip'; ignored
   Creating library xlsxwriter.lib and object xlsxwriter.exp
packager.c.obj : error LNK2019: unresolved external symbol fill_fopen_filefunc referenced in function lxw_packager_new
packager.c.obj : error LNK2019: unresolved external symbol zipOpen2 referenced in function lxw_packager_new
packager.c.obj : error LNK2019: unresolved external symbol zipOpen2_64 referenced in function _open_zipfile_win32
packager.c.obj : error LNK2019: unresolved external symbol zipOpenNewFileInZip4_64 referenced in function _add_file_to_zip
packager.c.obj : error LNK2019: unresolved external symbol zipWriteInFileInZip referenced in function _add_file_to_zip
packager.c.obj : error LNK2019: unresolved external symbol zipCloseFileInZip referenced in function _add_file_to_zip
packager.c.obj : error LNK2019: unresolved external symbol zipClose referenced in function lxw_create_package
packager.c.obj : error LNK2019: unresolved external symbol fill_win32_filefunc64W referenced in function _open_zipfile_win32

xlsxwriter.dll : fatal error LNK1120: 8 unresolved externals

ninja: build stopped: subcommand failed.
```
</details>

### Additional context

Comparing working output to broken output clearly shows pkgconfig causing dependencies to use different bits:

![Beyond Compare screenshot of CMake configure logs](https://github.com/user-attachments/assets/43ff1440-1df3-4c4e-a98a-dd81282cce90)

--- a/ports/libxlsxwriter/portfile.cmake
+++ b/ports/libxlsxwriter/portfile.cmake
@@ -17,6 +17,7 @@ endif()
 vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}"
     OPTIONS
+        -DCMAKE_DISABLE_FIND_PACKAGE_PkgConfig=ON
         -DUSE_SYSTEM_MINIZIP=1
         -DWINDOWSSTORE=${USE_WINDOWSSTORE}
 )

--- a/ports/libxlsxwriter/vcpkg.json
+++ b/ports/libxlsxwriter/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "libxlsxwriter",
   "version": "1.1.9",
+  "port-version": 1,
   "description": "Libxlsxwriter is a C library that can be used to write text, numbers, formulas and hyperlinks to multiple worksheets in an Excel 2007+ XLSX file.",
   "homepage": "https://github.com/jmcnamara/libxlsxwriter",
   "license": "BSD-2-Clause",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -5434,7 +5434,7 @@
     },
     "libxlsxwriter": {
       "baseline": "1.1.9",
-      "port-version": 0
+      "port-version": 1
     },
     "libxml2": {
       "baseline": "2.13.5",

--- a/versions/l-/libxlsxwriter.json
+++ b/versions/l-/libxlsxwriter.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "d19c223fca5963696284824fa0ebb515f87d6b2e",
+      "version": "1.1.9",
+      "port-version": 1
+    },
+    {
       "git-tree": "a4a8c37d3a9f01b4da4dd513178a92fc7b671cb8",
       "version": "1.1.9",
       "port-version": 0

